docs/components/output-mode-table.mdx

```diff
@@ -0,0 +1,6 @@
+| option    | description                              |
+| --------- | ---------------------------------------- |
+| full      | This is the default. Displays all output |
+| hash-only | Show only the hashes of the tasks        |
+| new-only  | Only show output from cache misses       |
+| none      | Hides all task output                    |
```

Comment:
MDX supports a ["partial" imported as a component][1], but I don't see any existing usages of this, so just calling out that this is a new pattern.

[1]: https://mdxjs.com/docs/using-mdx/#mdx-content